pkg/scheduler/server/api.go

```diff
@@ -142,78 +120,67 @@ func (s *Server) GetJob(ctx context.Context, req *schedulerv1pb.GetJobRequest) (
 	case <-s.readyCh:
 	}
 
-	jobName := req.GetJobName()
-	// TODO(artursouza): Use "FetchJob()" instead to read from db instead of running jobs only - once method exists.
-	job := s.cron.GetJob(jobName)
+	jobName, err := buildJobName(req.GetName(), req.GetMetadata())
+	if err != nil {
+		return nil, err
+	}
+
+	job, err := s.cron.Get(ctx, jobName)
+	if err != nil {
+		log.Errorf("error getting job %s: %s", jobName, err)
+		return nil, err
+	}
+
 	if job == nil {
 		return nil, fmt.Errorf("job not found: %s", jobName)
 	}
 
-	//ttl := ""
```

Comment:
do we not need to do any parsing on the ttl? I think I had it commented out and artur was going to add it back. just want to make sure we dont miss anything :)

Reply:
We don't, gets handled in the library.